src/main/java/org/folio/circulation/resources/DeclareClaimedReturnedItemAsMissingResource.java

```diff
@@ -23,42 +31,70 @@
 
 public class DeclareClaimedReturnedItemAsMissingResource extends Resource {
 
+  private static final String NOTE_MESSAGE = "Claimed returned item marked missing";
+  private static final String NOTE_DOMAIN  = "loans";
+
   public DeclareClaimedReturnedItemAsMissingResource(HttpClient client) {
     super(client);
   }
 
   @Override
   public void register(Router router) {
     new RouteRegistration("/circulation/loans/:id/declare-claimed-returned-item-as-missing", router)
-      .create(this::declareClaimedReturnedItemAsMissing);
+        .create(this::declareClaimedReturnedItemAsMissing);
   }
 
   private void declareClaimedReturnedItemAsMissing(RoutingContext routingContext) {
     final WebContext context = new WebContext(routingContext);
 
-    createRequest(routingContext)
-      .after(request -> processDeclareClaimedReturnedItemAsMissing(routingContext, request))
-      .thenApply(r -> r.toFixedValue(NoContentResponse::noContent))
-      .thenAccept(context::writeResultToHttpResponse);
+    createRequest(routingContext).after(request -> processDeclareClaimedReturnedItemAsMissing(routingContext, request))
+        .thenApply(r -> r.toFixedValue(NoContentResponse::noContent)).thenAccept(context::writeResultToHttpResponse);
```

Comment:
Why has this formatting changed to have multiple steps on the same line? 

Is it to comply with the standard line continuation guidance? 

If so, I think this is an exception to that guidance. Even though this is strictly one statement, I consider each step to be logically separate and find it much easier to read on separate lines. I find myself missing steps when they are on the same line.

What do you think?

Reply:
I don't know why these steps were consolidated, but I agree that they should be split to multiple lines.